### PR TITLE
addition of hana availability insights and performance insights for internal review

### PR DESCRIPTION
**Pull request by jbaskar**

### CODE REVIEW

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook [hunk 1]
```diff
@@ -58,7 +55,8 @@
             "isHiddenWhenLocked": true,
             "timeContext": {
               "durationMs": 86400000
-            }
+            },
+            "value": "PE1"
```
Comment (hirenchalodiya1) [inline Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:59]: Is this hard code value? (Looks SID to me)
Please confirm its not hard coded value here

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook [hunk 2]
```diff
@@ -68,15 +66,28 @@
             "isHiddenWhenLocked": true,
             "timeContext": {
               "durationMs": 86400000
-            }
+            },
+            "value": "SYSTEMDB"
```
Comment (hirenchalodiya1) [inline Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:70]: verify this one as well

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook [hunk 3]
```diff
@@ -68,15 +66,28 @@
...
           },
           {
             "id": "0b81c9d3-446c-4223-b424-f8168f2b5050",
             "version": "KqlParameterItem/1.0",
             "name": "TimePeriod",
             "type": 1,
             "isGlobal": true,
-            "isHiddenWhenLocked": true
+            "isHiddenWhenLocked": true,
+            "value": "2022-11-22T08:06:46.597Z - 2022-12-01T08:11:15.182Z"
```
Comment (hirenchalodiya1) [inline Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:90]: Time difference is hardcoded here?

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook [hunk 4]
```diff
@@ -186,7 +186,97 @@
...
+              "value::all"
+            ]
+          },
+          {
+            "id": "5f818549-c01f-4a9c-b804-d7cd18e67d60",
+            "version": "KqlParameterItem/1.0",
+            "name": "SAPComponent",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": "SAP Hana"
```
Comment (hirenchalodiya1) [inline Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:200]: Component name is `SAPComponent`
So wondering if this value will always be `SAP HANA`

**Review by hirenchalodiya1 — CHANGES_REQUESTED** [review]

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook (hunk 2 above)
Comment (jbaskar) [reply Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:70]: this does not affect the workbook as the value is always passed to it from parent template.

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook (hunk 3 above)
Comment (jbaskar) [reply Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:90]: same as previous comment. The value is always passed to the template from the parent workbook. having a default is inconsequential.

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook (hunk 4 above)
Comment (jbaskar) [reply Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:200]: same as previous comments.

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook [hunk 5]
```diff
@@ -281,9 +383,40 @@
...
+            "type": 11,
+            "isRequired": true,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[\r\n    {\"value\":\"VM\", \r\n    \"label\": \"Azure Infrastructure\"},\r\n    {\"value\":\"Process\",\r\n    \"label\":\"SAP Component\"}\r\n]",
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "value": "VM"
```
[the remark this answers is not in the document]
Comment (jbaskar) [reply Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:406]: no. value can also be "Process". Again, decided by the parent workbook.

File: Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook (hunk 1 above)
Comment (jbaskar) [reply Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook:59]: SID is always passed by the parent workbook so the value set in the template internally is irrelevant.

**Review by hirenchalodiya1 — APPROVED** [review]

**Pull request merged**